src/main/java/org/jenkinsci/plugins/vncrecorder/VncRecorderBuildWrapper.java

```diff
@@ -350,7 +350,7 @@ public FormValidation doCheckVncServ(@AncestorInPath AbstractProject<?,?> projec
 			}
 			//	    		List<String> com;
 			DescriptorImpl DESCRIPTOR = Hudson.getInstance().getDescriptorByType(DescriptorImpl.class);
-			String vnc2swf = Util.nullify(DESCRIPTOR.getVnc2swf());
+			String vnc2swf = Util.escape(Util.nullify(DESCRIPTOR.getVnc2swf()));
```

Comment:
This one makes sense as well.

Reply:
I even suggest to only do the escaping inside the errorWithMarkup as the escape is changing some of the characters you could potentially find in filePath, like space.

Using Util.xmlEscape is sufficient for "general" XSS protection like required here as it's only escaping the <> and &.

Reply:
Right, `Util#escape` is the weird one that converts newlines into `<br/>`, basically a "HTML as plain text" converter.